com.unity.cinemachine/Samples~/Shared Assets/Editor.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
```

Comment:
Please delete this file, it's not needed

Reply:
Fixed in latest commits